### PR TITLE
chore: update editorconfig to match the current project style

### PR DESCRIPTION
**Pull request by oltarasenko**

### CODE REVIEW

**Review by twilson63 — APPROVED** [review]
lgtm

**Comment by samcamwilliams** [comment]
Other than that flag, LGTM! Thanks

**Comment by twilson63** [comment]
@oltarasenko I think we need to get this merged so others can benefit of the correct editor config

File: .editorconfig [hunk 1]
```diff
@@ -9,9 +9,9 @@ end_of_line = lf
 
 trim_trailing_whitespace = true
```
Comment (twilson63) [inline .editorconfig:10]: @oltarasenko please adjust to `false`

**Pull request merged**